signin/signin.html

```diff
@@ -4,61 +4,51 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>sign in</title>
-    <link rel="stylesheet" href="signin.css" />
+    <link rel="stylesheet" href="../src/colors.css" />
+    <link rel="stylesheet" href="../src/global.css" />
+    <link rel="stylesheet" href="../src/reset.css" />
+    <link rel="stylesheet" href="./signin.css" />
   </head>
   <body>
-    <main class="container">
-      <div class="box">
-        <header>
-          <div>
-            <a href="/"
-              ><img src="/images/logo.svg" alt="logo" class="logo"
-            /></a>
+    <header>
+      <a href="/"><img src="../images/logo.svg" alt="logo" class="logo" /></a>
+      <p>
+        회원이 아니신가요?
+        <a href="../signup/signup.html" class="goSignUp">회원 가입하기</a>
+      </p>
+    </header>
+    <div class="signInBox">
+      <form action="/folder.html" onsubmit="return login()">
```

Comment:
return 따로 안하셔도 될 거에요!

```suggestion
      <form action="/folder.html" onsubmit="login()">
```